lib/shopify_app/configuration.rb

```diff
@@ -163,16 +163,18 @@ def post_authenticate_tasks
 
     private
 
-    def log_callback_controller_method_deprecation
+    def log_v23_deprecations
       return unless Rails.env.development?
 
       # TODO: Remove this before releasing v23.0.0
       message = <<~EOS
         ================================================
-        => Upcoming deprecation in v23.0:
+        => Upcoming deprecations in v23.0:
```

Comment:
```suggestion
        => Upcoming changes in v23.0:
```

? These are currently deprecated.